Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM python:2.7-slim-buster
+
+# Install dependencies
+RUN apt-get update && apt-get install -y \
+    gcc \
+    libxml2-dev \
+    libxslt-dev
+
+# Create and user user
+RUN useradd --create-home djangoplicityadm
```

Comment:
This Dockerfile would be only for development right?. Then managing users permissions inside the containers is not required, that's a security concern for production applications to disallow users inside containers to gain access to the host machine.

Reply:
Oooh, okay! I understand now, thanks!